test/common/testutil.cc

```diff
@@ -206,6 +207,8 @@ DirectoryEntry DirectoryEntryTestFactory::RegularFile(const string &name,
                                                       shash::Any hash) {
   DirectoryEntry dirent;
   dirent.mode_ = 33188;
+  dirent.uid_ = getuid();
```

Comment:
The changes in this file were intended to allow for unittesting against libcvmfs. This is currently unused. This is a result of not having actual files written out with the fake repository, which causes sync to fail.